testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/deployment/classloading/ear/subdeployments/SubDeploymentAvailableInClassPathTestCase.java

```diff
@@ -146,11 +151,11 @@ public static EnterpriseArchive createEarWithMultipleWars() {
      */
     @Test
     @OperateOnDeployment("ear-with-single-war")
-    public void testEjbClassAvailableInServlet() throws Exception {
-        final HttpClient httpClient = new DefaultHttpClient();
+    public void testEjbClassAvailableInServlet(@ArquillianResource(HelloWorldServlet.class) URL baseUrl) throws Exception {
+        final HttpClient httpClient = HttpClientBuilder.create().build();
```

Comment:
try (final CloseableHttpClient httpClient HttpClients.createDefault()){...

if we are fixing this lets fix it properly...